views/export/preview-pdf.pug

```diff
@@ -38,4 +38,5 @@ block content
         var maxNumberOfSongs = !{JSON.stringify(maxNumberOfSongs).replace(/<\//g, "<\\/")}
     script(src='/dist/javascripts/exportOption.js')
     script.
+        import m from 'mithril';
```

Comment:
this won't work because browsers don't support import natively yet (we gotta build it via gulp/parcel). Since this is all part of the export flow, we can work on improving that next.